src/main/java/teammates/ui/controller/ControllerServlet.java

```diff
@@ -126,7 +127,7 @@ public final void doPost(HttpServletRequest req, HttpServletResponse resp) throw
              * in GAE shutting down the instance.
              * Note that severe logs are sent by email automatically in the cron job auto/compileLogs.
              */
-            log.severe("Unexpected exception " + t.getMessage());
+            log.severe(TeammatesException.toStringWithStackTrace(t));
```

Comment:
Better. Can you also include a`"Unexpected exception caught by ControllerServlet : "` ? I would like an easy way to identify log messages originating from this location.